### ISSUE TITLE
Create an ownership method, to allow faction warfare

### ISSUE THREAD

**Issue by broomstickmk2** [seymourimadeit/guardvillager-suggestions#104]
Similar to mods like Overlord, it would be awesome to have some sort of an item that can be used to claim villager guards, so we can create armies.

**Comment by The-Artificer** [comment]
Agreed, some sort of player loyalty would be great, so they could act more like dogs, i.e teleport to the player if loaded, and attack whatever attacks the player to defend them- including other players. I wish there was more functionality to the follow so they could become a little more of an actual companion, and you could have a private army, rather than them just temporarily following you while you have the hero of the village effect, and using their normal targeting. Maybe if you give them something while you have hero of the village, like a totem of undying, then they will become 'hired' as a proper follower- or a 'tamed' mob in minecraft terms.

**Comment by The-Artificer** [comment]
It just feels really underwhelming that they are only temporary pseudo-followers that require you to have a potion effect, and that they will randomly pick any player with the effect to follow instead if they run past you, and that anyone with the effect can command. Especially when the idea of having NPC followers who you can equip with gear- including modded gear- is really neat. :/

**Comment by Firelancer120** [comment]
But they arent actual followers they are village guards.  If they are always off with you they arent protecting the village right
